app/jinja_filters.py

```diff
@@ -349,7 +323,9 @@ def __init__(self, detail_answer_field, detail_answer_schema):
 def map_checkbox_config(form, answer):
     options = form["fields"][answer["id"]]
 
-    return [CheckboxConfig(option, i, form, answer) for i, option in enumerate(options)]
+    return [
+        SelectConfig(option, idx, answer, form) for idx, option in enumerate(options)
```

Comment:
Do we want to keep it consistent and call it `index` instead of `idx` since SelectConfig takes `index` as an argument? Same on line 341.

Reply:
Updated